src/schemas/v2/combined.json

```diff
@@ -7774,74 +7958,41 @@
                   "title": "Parking arrangements",
-                  "baspiRequired": ["parkingArrangementsType"],
-                  "ta6Required": ["parkingArrangementsType"],
-                  "properties": {
-                    "parkingArrangementsType": {
-                      "baspiRef": "A1.6.0.1",
-                      "ta6Ref": "9.1.1",
-                      "piqRef": "A1.5.1",
-                      "title": "What type of parking arrangements are there?",
-                      "type": "string",
-                      "enum": [
-                        "Garage",
-                        "Allocated parking space",
-                        "Driveway",
-                        "On street",
-                        "Resident permit",
-                        "Metered parking",
-                        "Communal parking",
-                        "None",
-                        "Other"
-                      ]
-                    }
-                  },
-                  "discriminator": {
-                    "propertyName": "parkingArrangementsType"
-                  },
-                  "oneOf": [
-                    {
-                      "properties": {
-                        "parkingArrangementsType": {
-                          "enum": [
-                            "Garage",
-                            "Allocated parking space",
-                            "Driveway",
-                            "On street",
-                            "Resident permit",
-                            "Metered parking",
-                            "Communal parking",
-                            "None"
-                          ]
-                        }
-                      }
-                    },
-                    {
-                      "properties": {
-                        "parkingArrangementsType": {
-                          "enum": ["Other"]
-                        },
-                        "otherParkingType": {
-                          "baspiRef": "A1.6.0.2",
-                          "ta6Ref": "9.1.2",
-                          "title": "Please describe the other parking arrangements",
-                          "type": "string"
-                        }
-                      },
-                      "baspiRequired": ["otherParkingType"],
-                      "ta6Required": ["otherParkingType"]
-                    }
-                  ]
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "Allocated",
+                      "Communal",
+                      "Covered",
+                      "Garage",
+                      "Driveway",
+                      "Gated",
+                      "Off Street",
+                      "On Street",
+                      "Rear",
+                      "Private",
+                      "Disabled Parking Available",
+                      "No Disabled Parking Available",
```

Comment:
Very minor, I'd suggest deleting "No Disabled Parking Available". "Disabled Parking Available" not being selected infers the former.